src/org/parosproxy/paros/core/proxy/ProxyThread.java

```diff
@@ -321,14 +323,35 @@ private static void setErrorResponse(HttpMessage msg, String responseStatus, Exc
     private static void setErrorResponse(HttpMessage msg, String responseStatus, Exception cause, String errorType)
             throws HttpMalformedHeaderException {
         StringBuilder strBuilder = new StringBuilder();
-        strBuilder.append(errorType)
-                .append(" [")
-                .append(cause.getClass().getName())
-                .append("]: ")
-                .append(cause.getLocalizedMessage())
-                .append("\n\nStack Trace:\n");
-        for (String stackTraceFrame : ExceptionUtils.getRootCauseStackTrace(cause)) {
-            strBuilder.append(stackTraceFrame).append('\n');
+		
+        if (cause instanceof SSLException) {
+            strBuilder.append(Constant.messages.getString("network.ssl.error.connect"));
+            strBuilder.append(msg.getRequestHeader().getURI().toString()).append('\n');
+            strBuilder.append(Constant.messages.getString("network.ssl.error.exception") + cause.getMessage())
+                .append('\n');
+            strBuilder.append(Constant.messages.getString("network.ssl.error.exception.rootcause")
+                + ExceptionUtils.getRootCauseMessage(cause)).append('\n');
+            strBuilder.append(Constant.messages.getString("network.ssl.error.help",
+                Constant.messages.getString("network.ssl.error.help.url")));
+            
+            log.warn(strBuilder.toString());
+            if (log.isDebugEnabled()) {
+                log.debug(cause, cause);
+            	strBuilder.append("\n\nStack Trace:\n");

```

Comment:
Could use the same indentation char as before (space), same for surrounding statements.